source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,57 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+    template<typename Policy, typename KeyIt, typename ValueIt,
+        typename Comparator = std::less<typename std::iterator_traits<InputIt>::value_type>>
+    void
+    sort_by_key(Policy&& policy, KeyIt keys_first, KeyIt keys_last,
+        ValueIt values_first,
+        Comparator comp = std::less<typename std::iterator_traits<InputIt>::value_type>());
```

Comment:
```suggestion
    template<typename Policy, typename KeyIt, typename ValueIt,
        typename Comparator = std::less<typename std::iterator_traits<KeyIt>::value_type>>
    void
    sort_by_key(Policy&& policy, KeyIt keys_first, KeyIt keys_last,
        ValueIt values_first,
        Comparator comp = std::less<typename std::iterator_traits<KeyIt>::value_type>());
```

Similar change required for the signature below.  
I would prefer `{}` or `Comparator{}` as the default parameter to remove confusion, but I think this was already discussed, and intended to change together with the other algorithms.

Reply:
Applied for both sort algorithms.

Reply:
> I would prefer {} or Comparator{} as the default parameter to remove confusion, but I think this was already discussed, and intended to change together with the other algorithms.

Yes, it was discussed with an agreement to use `()` for the alignment with other algorithms on this page.